.devcontainer/devcontainer.json

```diff
@@ -5,53 +5,32 @@
     "build": {
         "dockerfile": "Dockerfile",
         "args": {
-            // Update the VARIANT arg to pick a Java version: 11, 15
-            "VARIANT": "17",
-            // Options
-            "INSTALL_ZSH": "false",
-            "UPGRADE_PACKAGES": "false",
-            "ENABLE_NONROOT_DOCKER": "true",
-            "USE_MOBY": "true",
-            // Version of the Trino CLI to install
-            "TRINO_VERSION": "400"
+            "BASE_IMAGE_TAG": "420_feature_new-dev-environment"
```

Comment:
Does this imply trino version 420? Not sure if it matters for this dev environment but I was wondering if it's actually tested with 452

Reply:
Yeah, this is one of the spots where I will change the tag once we have a valid build on `cccs-main`. Second PR coming up shortly after this one is merged.